src/Login/ProtectedContent.jsx

```diff
@@ -61,7 +52,6 @@ export function intersection(aList, bList) {
  * @param history from react-router
  * @param match from react-router.match
  * @param public default false - set true to disable auth-guard
- * @param background passed through to <Box background> wrapper for page.jsx-level background
```

Comment:
do we need this param for anything?

Reply:
I did some global searching, and found that `background-color` is only different for `ExplorerComponent` and was passed via `background` variable. So to make code cleaner I set `background-color`  inside `ExplorerComponent.less`, and remove this `background` parameter from `ProtectedContent` component.

Reply:
awesome, makes sense to me